app/helpers/finance_helper.rb

```diff
@@ -9,6 +23,19 @@ def float_to_percentage(number)
     number_to_percentage(number * 100, precision: 0)
   end
 
+  def aggregated_payment(breakdown_array)
+    breakdown_array.map { |breakdown|
+      breakdown[:service_fees][:monthly] + breakdown[:output_payments][:subtotal]
+    }.inject(&:+)
```

Comment:
Can `breakdown_array` be empty here? You may want to default to `0`, otherwise this would return `nil`.

```suggestion
    }.inject(0, &:+)
```

Reply:
Don't think it could be but that does to hurt